src/towncrier/_shell.py

```diff
@@ -11,9 +11,12 @@
 from .build import _main as _build_cmd
 from .check import _main as _check_cmd
 from .create import _main as _create_cmd
+from ._version import __version__
 
 
 @click.group(cls=DefaultGroup, default="build", default_if_no_args=True)
+@click.version_option(__version__.public())
+@click.pass_context
```

Comment:
Is this needed? I think that this is why the test fails.

I see the tests are green even if this is removed

Reply:
Ahh, when I remove this line it works! :+1: